projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-v1-4-endpoint.php

```diff
@@ -120,6 +120,7 @@
 			'wpcom_newsletter_categories'             => '(array) Array of post category ids that are marked as newsletter categories',
 			'wpcom_newsletter_categories_enabled'     => '(bool) Whether the newsletter categories are enabled or not',
 			'sm_enabled'                              => '(bool) Whether the newsletter Subscribe Modal is enabled or not',
+			'sb_post_end_enabled'                     => '(bool) Whether adding Subscribe block at the end of each post is enabled or not',
```

Comment:
While there are mixed bag of examples here and elsewhere already, let's look into prefixing the option with `jetpack_` — I believe that has some benefits as we get some option management for free, and might be more up-to-date how we do options in Jetpack these days. Crew team would know the best.

Reply:
Discussed it with Crew folks (p1707222683709909-slack-CBG1CP4EN) and we decided to add the prefix. Thanks for spotting this! Should be fixed already.